pkg/server/handler/v1alpha1/webhook.go

```diff
@@ -143,6 +143,9 @@ func createWorkflowRun(tenant, wftName string, data *scm.EventData) error {
 	wfr := &v1alpha1.WorkflowRun{
 		ObjectMeta: metav1.ObjectMeta{
 			Name: tag,
+			Annotations: map[string]string{
+				common.AnnotationTrigger: string(data.Type),
```

Comment:
For other types of wfr, still need trigger annotation, IMO.

Reply:
Have added trigger for workflowruns triggered by cron timer